src/schema/metadata/BackgroundSuppressionNumberPulses.yaml

```diff
@@ -0,0 +1,8 @@
+---
+name: BackgroundSuppressionNumberPulses
+description: |
+  The number of background suppression pulses used.
+  Note that this excludes any effect of background suppression pulses applied
+  before the labeling.
+type: integer
```

Comment:
This is a number in the specification table. I think an integer is more accurate.

Reply:
Would a float even make sense for this field? If no, we can consider it a bugfix to change the specification table to say integer, instead of number?

Reply:
It doesn't seem like it would be possible to have a fractional pulse, but I just wanted to make sure.